tests/unit/IncorporationDateTime.spec.ts

```diff
@@ -222,7 +222,7 @@ describe('Incorporation Date Time', () => {
     const radioIsFutureEffective = radioInput.at(1)
     await radioIsFutureEffective.trigger('click')
 
-    const minTime = wrapper.vm.minTime
+    const minTime = wrapper.vm.minTime()
```

Comment:
This passes TS checks? I've always had to cast vm to `any` because TS could never find the method or property on vm. If there's a setting that allows this to work correctly then we should port that setting to Filings UI.

Reply:
I wonder if this works because you are using wrapperFactory() ?

Reply:
Yes, that's why. You declared wrapperFactory as `any` :)

Reply:
Cam, go ahead and resolve this conversation once you've read these comments.

Reply:
Interesting! I'm seeing no errors, nor did i change any of the TS configurations 🤔

Reply:
Could be because the factory method is described as `any`?

Reply:
We must have been commenting at the same time :)